src/Hng.Application.Test/Features/Organization/CreateOrganizationShould.cs

```diff
@@ -0,0 +1,84 @@
+﻿using AutoMapper;
+using Hng.Application.Features.Organisations.Commands;
+using Hng.Application.Features.Organisations.Dtos;
+using Hng.Application.Features.Organisations.Handlers;
+using Hng.Application.Features.Organisations.Mappers;
+using Hng.Domain.Entities;
+using Hng.Infrastructure.Repository.Interface;
+using Moq;
+using Xunit;
+
+namespace Hng.Application.Test.Features.Organisations
+{
+    public class CreateOrganizationCommandHandlerTests
```

Comment:
Your test file and your test class have different names. Fix this.

Reply:
DONE